templates/swagger-typescript-api/http-client.ejs

```diff
@@ -38,7 +38,6 @@ export enum ContentType {
 
 export class HttpClient<SecurityDataType = unknown> {
     public instance: AxiosInstance;
-    public prerequest: Promise<any> = Promise.resolve();

```

Comment:
This now more-or-less the same as the default http-client.ejs used by `swagger-typescript-codegen` aside from the tweaked baseUrl